pkg/api/handlers/utils/images.go

```diff
@@ -24,6 +25,14 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+type BuildResponse struct {
+	Stream string                 `json:"stream,omitempty"`
+	Error  *jsonmessage.JSONError `json:"errorDetail,omitempty"`
+	// NOTE: `error` is being deprecated check https://github.com/moby/moby/blob/master/pkg/jsonmessage/jsonmessage.go#L148
```

Comment:
Same comment as above:  Why use/introduce a new struct with already deprecated fields?

Reply:
@rhatdan that's still open.